core/cli/cli.hpp

```diff
@@ -136,4 +137,123 @@ namespace fc::cli {
   using Group = Empty;
 
   struct ShowHelp {};
+
+  class TableWriter {
```

Comment:
Look at Ruslan's Suggestion

Reply:
Yes, fixed

Reply:
It's also about [refactoring/simplifying `TableWriter`](https://github.com/filecoin-project/cpp-filecoin/blob/suggest/table_writer/core/common/table_writer.hpp)

Reply:
Yes, I will look and refactoring TableWriter after a while